src/main/java/org/testng/internal/Configuration.java

```diff
@@ -117,4 +124,15 @@ public IExecutorFactory getExecutorFactory() {
   public boolean alwaysRunListeners() {
     return alwaysRunListeners;
   }
+
+  @Override
+  public void setInjectorFactory(IInjectorFactory factory) {
+    this.injectorFactory = factory;
+
```

Comment:
blanck line

Reply:
Will fix this as well, once I update this PR after going home.